seedlet/template-parts/post/author-bio.php

```diff
@@ -2,16 +2,14 @@
 /**
  * The template for displaying Author info
  *
- * @package WordPress
- * @subpackage Seedlet
+ * @package Seedlet
  * @since 1.0.0
  */
 
 if ( (bool) get_the_author_meta( 'description' ) ) : ?>
 <div class="author-bio">
 	<?php
-	_e(__( 'Published by', 'seedlet' ));
-	?>
+	_e(__( 'Published by', 'seedlet' )); ?>
```

Comment:
The reviewer cited this line as a problem, but I'm not sure why.

Reply:
Looks like we're unnecessarily using two translation functions there. `__()` will return the translated string, while `_e()` will echo it. In this case, we should just go with `_e()`.